packages/firestore/src/core/sync_engine.ts

```diff
@@ -410,12 +412,15 @@ export class SyncEngine implements RemoteSyncer, SharedClientStateSyncer {
   }
 
   // PORTING NOTE: Multi-tab only
-  applyPendingBatch(batchId: BatchId): Promise<void> {
-    // TODO(multitab): Implement applyPendingBatch
-    throw new FirestoreError(
-      Code.UNIMPLEMENTED,
-      'applyPendingBatch not implemented'
+  async applyPendingBatch(batchId: BatchId): Promise<void> {
```

Comment:
I think I like applyBatchFromPersistence().  But maybe just add a TODO on SharedClientStateSyncer for now.  It may make sense to rename all of those methods together (I wonder if adding "FromPersistence" suffixes on all of the methods might possibly make sense).

Reply:
Added this as a TODO.